test/UnitTestREADME.txt

```diff
@@ -0,0 +1,20 @@
+Creating and adding new unit tests - 
+
```

Comment:
But this is just for wav test, creating other unit tests means covering other functionalities as well.

Reply:
Okay, I'll add tests for other functionalities as well

Reply:
Since it's just for wav test, it would be better if you change the title to "Add new wav tests".
Also, changing the file name to "README.txt" or "README.md" (markdown) would be better since GitHub can automatically shows the content of it when you open the folder in the web.

Reply:
I'll be adding more unit tests shortly, apart from the wav tests.